sanssouci/post_hoc_bounds.py

```diff
@@ -72,18 +92,25 @@ def curve_max_fp(p_values, thr):
     Upper bound for the number of false discoveries among most
     significant items.
 
-    * Parameters:
-        - p_values: A 1D numpy array containing all $p$ p-values,
-                    sorted non-decreasingly
-        - thr: A 1D numpy array  of $K$ JER-controlling thresholds,
-               sorted non-decreasingly
-    * Returns:
-        - A vector of size p giving an joint upper confidence bound on
-          the number of false discoveries among the $k$ most significant
-          items for all k in \{1,\ldots,m\}
-    * Reference:
-        - Blanchard, G., Neuvial, P., & Roquain, E. (2020).
-        Post hoc confidence bounds on false positives using reference families.
+    Parameters
+    ----------
```

Comment:
I think that you should leave a blank after.

Reply:
You mean a blank *line* after the '----------'?

Reply:
Exactly.

Reply:
OK thanks. @nicolas-enjalbert  will take care of this when he's back (ie in April).